src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -248,5 +252,57 @@ public void Dispose()
                 disposed = true;
             }
         }
+
+        [DllImport("advapi32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool LogonUser(String lpszUsername, String lpszDomain, String lpszPassword,
+       int dwLogonType, int dwLogonProvider, out SafeAccessTokenHandle phToken);
+
+        internal async Task HandleValidateWindowsAccountRequest(
+            ValidateWindowsAccountRequestParams parameters,
+            RequestContext<ValidateWindowsAccountResult> requestContext)
+        {
+            var domainUserRegex = new Regex(@"^[A-Za-z0-9\\\._-]{7,}$");
```

Comment:
Why min 7 characters?

Something like this seems better : 

`^(?<domain>[A-Za-z0-9\._-]*)\\(?<username>[A-Za-z0-9\._-]*)$`

(and then you can use the groups from the result instead of substring below for the domain/username.